pyquil/api/_wavefunction_simulator.py

```diff
@@ -66,7 +77,7 @@ def __init__(
         self._qvm_client = QVMClient(client_configuration=client_configuration, request_timeout=timeout)
 
     @_record_call
-    def wavefunction(self, quil_program: Program, memory_map: Any = None) -> Wavefunction:
+    def wavefunction(self, quil_program: Program, memory_map: Dict[str, List[Union[int, float]]] = {}) -> Wavefunction:
```

Comment:
Be wary of using mutable datatypes for initial values in function parameters. Any mutation will persist between function invocations.

Reply:
Oh that's weird. Good to know. I'll make it optional then

Reply:
Fixed! Thanks again